shared/pages/PartialClosure/PartialClosure.js

```diff
@@ -402,6 +427,10 @@ export default class PartialClosure extends Component {
       }))
     }
 
+    if (constants.coinsWithDynamicFee.includes(getCurrency) && BigNumber(maxAllowedGetAmount).isGreaterThan(0)) {
```

Comment:
maxAllowedGetAmount уже BigNumber:
https://github.com/swaponline/swap.react/pull/1689/files#diff-2d0170c939b16e092aeccad030b85b8eR390
Значит можно делать сразу так:
```suggestion
    if (constants.coinsWithDynamicFee.includes(getCurrency) && maxAllowedGetAmount.isGreaterThan(0)) {
```
Вот тут правильно сделал:
https://github.com/swaponline/swap.react/pull/1689/files#diff-2d0170c939b16e092aeccad030b85b8eR659